src/pages/ServiceGraph/SummaryPanelNode.tsx

```diff
@@ -14,6 +14,7 @@ import { Icon } from 'patternfly-react';
 import { authentication } from '../../utils/Authentication';
 import { Link } from 'react-router-dom';
 import { shouldRefreshData } from './SummaryPanelCommon';
+import { HealthIndicator, DisplayMode } from '../../components/ServiceHealth/HealthIndicator';
```

Comment:
Adding the icon an popover to the summaries is good,  I had added this in my Overview POC as well.  We could consider adding one to the whole-graph summary, as an aggregate of the graph services (not in this PR) .

Reply:
+1 I thought about something similar too :)